app/assets/stylesheets/modules/filters-popups.css.scss

```diff
@@ -45,19 +53,31 @@
   box-shadow: 1px 1px 4px 0 rgba(160, 192, 290, 0.5);
   border: none;
   color: $dark-blue;
+  cursor: pointer;
+  flex: 1 0 48%;
   font-size: $tiny-font;
   font-weight: $font-stack-regular;
-  width: 126px;
-  padding: 6px 0;
+  padding: 8px 12px;
   margin-bottom: 12px;
+
+  &.active {
+    background: $blue;
+    color: white;
+    font-weight: $font-stack-bold;
+  }
+
+  &:nth-child(2n) {
+    margin-left: 3%;
+  }
 }
 
-.filter-popup-button--more {
+.filter-popup-secondary-button {
```

Comment:
changed the name bc it's not actually a modifier class but a separate one = isn't added in addition to the base class but instead of it

Reply:
Thanks for explaining!